files/class-vip-filesystem.php

```diff
@@ -11,6 +11,21 @@ class VIP_Filesystem {
 	 */
 	const PROTOCOL = 'vip';
 
+	/**
+	 * The name of the scheduled cron event to update attachment metadata
+	 */
+	const CRON_EVENT_NAME = 'update_attachment_filesizes';
```

Comment:
Just noticed that we should prefix this with `vip_`

```suggestion
	const CRON_EVENT_NAME = 'vip_update_attachment_filesizes';
```